docs/pipelineruns.md

```diff
@@ -676,6 +676,7 @@ Conditions:
   Type:                  Succeeded
 Skipped Tasks:
   Name:       skip-this-task
+  Reason:     WhenExpressionsSkip
```

Comment:
please update lines 666-668 as well to indicate that a Reason field will be added

Reply:
Done.